sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -286,3 +286,19 @@ ALTER TABLE `form_questionnaire_assessments` CHANGE `user` `user` VARCHAR(255) N
 #IfMissingColumn module_configuration date_created
 ALTER TABLE `module_configuration` ADD COLUMN `date_created` DATETIME DEFAULT NULL COMMENT 'Datetime the record was created';
 #EndIf
+
+#IfRow2D globals gl_name login_page_layout gl_value center
+UPDATE `globals` SET `gl_value` = 'login/layouts/vertical_box.html.twig' WHERE `gl_name` = 'login_page_layout' AND `gl_value` = 'center';
+#EndIf
+
+#IfRow2D globals gl_name login_page_layout gl_value left
+UPDATE `globals` SET `gl_value` = 'login/layouts/horizontal_box_left_logo.html.twig' WHERE `gl_name` = 'login_page_layout' AND `gl_value` = 'left';
+#EndIf
+
+#IfRow2D globals gl_name login_page_layout gl_value right
+UPDATE `globals` SET `gl_value` = 'login/layouts/horizontal_band_right_logo.html.twig' WHERE `gl_name` = 'login_page_layout' AND `gl_value` = 'right';
+#EndIf
+
+#IfNotRow globals gl_name secondary_logo_position
+INSERT INTO `globals` gl_name='secondary_logo_position', gl_value='second';
+#EndIf
```

Comment:
remove above 3 lines. new globals are automatically added with default in globals.inc.php during install/upgrade, so no need to put here.

Reply:
Done